src/main/java/org/zalando/nakadi/repository/TopicRepository.java

```diff
@@ -19,6 +19,24 @@
 
 public interface TopicRepository {
 
+    class TimelineAndPartition {
```

Comment:
Does the partition belongs to timeline? If yes - I think it's better to name the class `TimelinePartition`.